content/zh/docs/releases/supported-releases/index.md

```diff
@@ -48,7 +48,7 @@ test: n/a
 | master          | 否，仅限开发 | - | - | - | - |
 | 1.15            | 是                   | 2022 年 8 月 31 日 | ~ 2023 年 3 月（预期）  | 1.22, 1.23, 1.24, 1.25         | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21  |
 | 1.14            | 是                   | 2022 年 5 月 24 日 | ~ 2023 年 1 月（预期）  | 1.21, 1.22, 1.23, 1.24         | 1.16, 1.17, 1.18, 1.19, 1.20        |
-| 1.13            | 是                   | 2022 年 2 月 11 日 | ~ 2022 年 10 月（预期） | 1.20, 1.21, 1.22, 1.23         | 1.16, 1.17, 1.18, 1.19              |
+| 1.13            | 是                   | 2022 年 2 月 11 日 | ~ 2022 年 10 月 12 日（预期） | 1.20, 1.21, 1.22, 1.23         | 1.16, 1.17, 1.18, 1.19              |
```

Comment:
It's wrong. Please really check it by following the en version. 1.13 has been released months ago. It shall not still be `expected`!

Reply:
<img width="880" alt="image" src="https://user-images.githubusercontent.com/76980726/192097033-14ec0b32-3400-4cbf-9685-95df1a5081a1.png">
yes